bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -51,6 +63,10 @@ limitations under the License.
           <groupId>io.opencensus</groupId>
           <artifactId>*</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
I think you can remove the exclusions for now?

Reply:
Yes, thanks for catching that!

Done with https://github.com/googleapis/java-bigtable-hbase/pull/2775/commits/09cfa0dab85aea96019687c40f6bd9e3d4b79fee